### PR TITLE
LPS-138087 Autogenerate get...Pages tests for top level entities

### PR DESCRIPTION
**Pull request by javierdearcos**

### CODE REVIEW

**Comment by javierdearcos** [comment]
ci:test:relevant

**Comment by javierdearcos** [comment]
ci:test:relevant

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:test:sf

**Comment by javierdearcos** [comment]
ci:test:relevant

**Review by nhpatt — APPROVED** [review]
topLevel is a strange word. I guess that you mean pages that do not depend in one parameter as siteId or something like that, right? Maybe root?

I would leave like this because Brian will have his preference and it's not important though.

**Review by nhpatt — APPROVED** [review]

File: modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (nhpatt) [inline modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/pom.xml:3]: I don't think this should be uploaded.

**Review by nhpatt — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/pom.xml (hunk 1 above)
Comment (javierdearcos) [reply modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/pom.xml:3]: Good catch! Sorry, I will remove it now

**Review by nhpatt — APPROVED** [review]

**Comment by nhpatt** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:test:relevant

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:stop

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:reevaluate:1277401_142

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
ci:forward

**Comment by javierdearcos** [comment]
Sending here: https://github.com/liferay-frontend/liferay-portal/pull/1454

**Pull request closed without merging**